java/odsa-spring/src/main/java/com/oracle/dev/jdbc/odsa/spring/Employee.java

```diff
@@ -0,0 +1,118 @@
+/*
+  Copyright (c) 2021, 2022, Oracle and/or its affiliates.
```

Comment:
Change year to 2024

Reply:
Ok, I did it for all classes with the license note and updated them to 2024 as you requested

Reply:
@Kuassim please review, approve and notify Jean per the process (2nd level approval). Thanks!